src/app/components/authenticated-v2/settings/desired-exotic-selection/desired-exotic-selection.component.html

```diff
@@ -37,11 +37,29 @@
       class="exoticIcon {{ selectedExotics.indexOf(exotic.item.hash) > -1 ? 'selected' : '' }}"
       src="https://www.bungie.net/{{ exotic.item.icon }}"
       [exoticTooltip]="exotic.item"
-      [class.disabled]="!exotic.inInventory"
-      (click)="exotic.inInventory && selectExotic(exotic.item.hash, $event)" />
+      [class.disabled]="
```

Comment:
Very good thinking.
I would like to also see a slight visual difference between:
Setting disabled:
- I have it in the vault (yellow)
- I do not have it in the vault (disabled)
If setting enabled:
- I have it in the vault (yellow)
- I do not have it in the vault but unlocked: slightly different color (maybe a css filter?)
- I do not have it unlocked at all (disabled)
That way you can easily see "yeah, this is different" :)